components/nimbus/tests/test_message_helpers.rs

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Testing get_experiment_branch semantics.
+
+mod common;
+#[allow(unused_imports)]
+#[allow(unused_attributes)]
+#[macro_use]
+use nimbus::error::Result;
+
+#[cfg(feature = "rkv-safe-mode")]
```

Comment:
out of curiosity, why is this necessary? I know it's in the other test files, but does it have any implication on targeting tests?

Reply:
I _think_ it's because we actually use a DB in testing. I don't know definitively, except it doesn't work without it. It's cargo cult by this time.